lib/asana/resources/story.rb

```diff
@@ -19,18 +19,22 @@ class Story < Resource
 
       attr_reader :created_by
 
-      attr_reader :hearted
+      attr_reader :liked
```

Comment:
And *this* is why asana-api-meta makes me cry and want to kick a pigeon.

Renaming hearts to likes https://github.com/Asana/asana-api-meta-internal/commit/e4e82bd3d1db02c5cccdfc219388828932a74778

has only four user-facing effects:
* Adds the "(num_)?lik(es|ed)" fields from our docs. This is good.
* Removes (num_)?heart(s|ed) fields to our docs. This is good.
* Adds the "(num_)?lik(es|ed)" fields as allowable attributes on our Ruby client library. This is good.
* Removes (num_)?heart(s|ed) fields from our Ruby client library. This is a breaking change.

The client library does "magic" to set these attributes here: https://github.com/Asana/ruby-asana/blob/master/lib/asana/resource_includes/resource.rb#L20, which only sets the attributes on this resource that are gettable. **Which no longer includes hearts if we merge this**.

[and, come to think of it, custom field settings may barf on this too because they're nested JSON under the hood, or have something gnarly like `client.Tasks.custom_field["enum_value"]` (string index into json) where everywhere else in the library would lead you to believe it'd be `client.Tasks.custom_field.enum_value`, (attribute access) like `client.Tasks.name`. What extra sucks is, looking into it, we use middleware called "faraday_middleware/multi_json" to do the parsing. This delegates to "whatever JSON parser you have installed". This means that if the underlying JSON parser returns a Hash, you must use quotes with square brackets. If it returns something like ActiveSupport::HashWithIndifferentAccess, **either a string index or dot notation is fine**] 😭

In other words, resources in this library mean something like a hybrid between what in Java is called a "resource" and a "model". They have attributes on them, like Java, and methods that call down, with reflection to fetch / instantiate _themselves_ rather than returning JSON.

Boyohboy I can't wait until we update these with a single designed architecture from openapi.

I'll create a fix in asana-api-meta and issue a new commit to this PR; the net effect is that we either have to patch asana-api-meta or we publicly expose in our API docs that hearts and likes are both valid, but PLEASE DON'T USE HEARTS THEY ARE DEPRECATED (this is what I'm going to do).